sros/docker/launch.py

```diff
@@ -629,22 +654,22 @@ def bootstrap_config(self):
 class SROS_lc(SROS_vm):
     """Line card for distributed VSR-SIM"""
 
-    def __init__(self, variant, conn_mode, num_nics, slot=1):
-        # lc - line card. role is used to create a separate overlay image name
+    def __init__(self, lc_config, conn_mode, num_nics, slot=1,  nic_eth_start=1):
+        # cp - control plane. role is used to create a separate overlay image name
```

Comment:
this belongs to SROS_lc, so the comment should be about `lc` part, not `cp`

Reply:
Well noted, I will update on my next PR